guest-tools/image/create-td-image.sh

```diff
@@ -219,6 +219,7 @@ setup_guest_image() {
     # export environment variables to guest
     # all environment variables with prefix : TDX_GUEST_SETUP_
     declare -px | grep TDX_GUEST_SETUP_ > ${CURR_DIR}/tdx-guest-setup-env
+    declare -px | grep INSTALL_ATTESTATION >> ${CURR_DIR}/tdx-guest-setup-env
```

Comment:
Thanks @bktan8 , it is great, We already have a mechanism to pass all environment variables with name TDX_GUEST_SETUP_* , I propose we use this channel to pass the attestation installation configuration, see my proposal here : https://github.com/bktan8/tdx/pull/1